app/controllers/stats_controller.rb

```diff
@@ -21,41 +23,47 @@ def sellers_total
     Seller.all.count
   end
 
-  def progress_bar_totals
-    query = ActiveRecord::Base.connection.execute($april_progress_bar_query)
-
-    total = 0
-    query.each do |type|
-      total += type['sum'] unless type['sum'].nil?
-    end
-
-    total
-  end
-
-  def num_days_remaining
-    currentDate = Time.now
-    endDate = Date.new(2021, 4, 26)
-    remainingDays = (endDate.to_date - currentDate.to_date).round
-    return 0 if remainingDays <= 0
-
-    (endDate.to_date - currentDate.to_date).round
-  end
-
   def transaction_totals
     Item.all.count
   end
 
   def show(donation_totals, sellers_total, transaction_totals, gam_count)
-    donation_totals = '$%s' % ActionController::Base.helpers.number_with_precision(donation_totals, precision: 0, delimiter: ',')  # "$10,000"
+    outside_db_gam_contributions = 0
+    outside_db_fundaising_contributions = 0
+    outside_db_fundaising_foodcrawl_raised = 36, 573
+    outside_db_luc_raised = 47, 689
```

Comment:
these commas make me nervous, maybe `rubocop -a` these? I think the ruby way is `47_689`